app/controllers/users_controller.rb

```diff
@@ -14,7 +14,9 @@ def create
     using_recaptcha = !params[:spamaway] && Rails.env == "production"
     recaptcha = verify_recaptcha(model: @user) if using_recaptcha
     @spamaway = Spamaway.new(spamaway_params) unless using_recaptcha
-    if ((@spamaway&.valid?) || recaptcha) && @user.save
```

Comment:
I guess o told you to shift this change to a new pr. This is not related to the current issue.
We aim at solving one unit of functionality in one pr

Reply:
Can you remove this file as it doesn't have any significant changes?